packages/@react-spectrum/breadcrumbs/docs/Breadcrumbs.mdx

```diff
@@ -22,6 +22,7 @@ import {View} from '@react-spectrum/view';
 
 ---
 category: Navigation
+keywords: [collections]
```

Comment:
i know the category is navigation, should that also be a keyword?

Reply:
each page `Category` is autofilled into the keywords list so that is covered without explicitly adding it

Reply:
items? dropdown?